src/interfaces/chain.h

```diff
@@ -138,6 +139,9 @@ class Chain
 
     //! Get virtual transaction size.
     virtual int64_t getVirtualTransactionSize(const CTransaction& tx) = 0;
+
+    //! Check if transaction is RBF opt in.
+    virtual RBFTransactionState isRBFOptIn(const CTransaction& tx) = 0;
```

Comment:
Could return the status string and thus avoid the include / simplify the interface.